pkg/client/client.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/acorn-io/acorn/pkg/streams"
 	"github.com/acorn-io/acorn/pkg/system"
 	"github.com/acorn-io/baaah/pkg/restconfig"
+	_ "github.com/golang/mock/mockgen/model"
```

Comment:
Why is this imported here? Is it so that package's init function runs?

Reply:
good catch, removed